remote-trait-object/src/context.rs

```diff
@@ -46,21 +46,23 @@ impl Context {
     }
 
     pub fn get_port(&self) -> Weak<dyn Port> {
-        Arc::downgrade(&self.port.clone().unwrap()) as Weak<dyn Port>
+        Arc::downgrade(&self.port.clone().expect("It becomes None only when the context is dropped.")) as Weak<dyn Port>
     }
 
     pub fn disable_garbage_collection(&self) {
-        self.port.as_ref().unwrap().set_no_drop();
+        self.port.as_ref().expect("It becomes None only when the context is the context is dropped.").set_no_drop();
```

Comment:
`when the context is the context is dropped.` -> `when the context is dropped."`

Reply:
fixed